scripts/check-dependencies.js

```diff
@@ -140,6 +140,11 @@ function main() {
   let success = true;
   const dependencyMaps = [];
   for (const packageJsonPath of getAllPackageJsonPaths()) {
+    if (!fs.existsSync(packageJsonPath)) {
```

Comment:
If a cache directory is set, travis creates it. This led to this script crashing because of `buidler-etherscan`'s `package.json` not being present. 

This change just prints a warning in those cases.

Reply:
Could you fix the typos in this file?
Like:
```
checkPerrDepedencies
pacakges
veresionSpeck
```